images/dcos/Makefile

```diff
@@ -18,10 +18,9 @@ GIT_REF_MINOR = $(lastword $(subst -, ,$(GIT_REF)))
 GIT_REF_MAJOR = $(firstword $(subst -, ,$(GIT_REF)))
 ifneq ($(wildcard $(BUILD_DIR)),)
 	UNCLEAN_INFIX := -unclean
-	UNCLEAN_PREFIX := $(USER)-
```

Comment:
could we do something like this instead? `UNCLEAN_PREFIX := $(shell echo $${USER:-nobody})-`